README.adoc

```diff
@@ -471,6 +471,79 @@ To remove the image, run the following command:
 docker rmi openliberty-getting-started:1.0-SNAPSHOT
 ```
 
+// =================================================================================================
+// Developing the application in a Docker container
+// =================================================================================================
+
+== Developing the application in a Docker container
+
+// File 0
+server.xml
+[source, xml, linenums, role='code_column']
+----
+include::finish/src/main/liberty/config/server.xml[]
+----
+
+The Open Liberty Maven plug-in includes a `devc` goal that simplifies developing
+your application in a Docker container. This goal builds a Docker
+image, mounts the required directories, binds the required ports, and then runs
+the application inside of a container. This development mode, known as dev mode, also listens for
```

Comment:
Here,  dev mode in a container is presented as a separate dev mode ("this dev mode")  from what is discussed earlier in the guide- but is it really a different mode?  Wouldn't either example be an instance of the same "dev mode" functionality? There are two goals (dev vs devc), but they enable the same basic functions. In the docs, it is presented as "container support for dev mode" https://www.openliberty.io/docs/21.0.0.1/development-mode.html#_container_support_for_dev_mode. Similar for the maven doc: https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md

I think this might be confusing for someone working through the entire guide. If dev mode, in and of itself, is different here than earlier in the guide, how and why? If it is not different, and is simply the same function being run by a different Maven goal, I think you could rewrite this to reflect that. Assume the reader has already been introduced to dev mode earlier in the guide, then explain how they can also run dev mode in a docker container. 


Furthermore, I'd recommend using "dev mode" throughout the guide, rather than "development mode". Alasdair asked that we make that change in the doc and it is consistent with documentation in the Git repos.